thoth/common/openshift.py

```diff
@@ -980,6 +980,80 @@ def get_package_extract_template(self):
 
         return self._get_template("template=package-extract")
 
+    def schedule_package_analyzer(
+        self,
+        package_name: str,
+        package_version: str,
+        index_url: str,
+        output: str,
+        debug: bool = False,
+        job_id: str = None,
+    ) -> str:
+        """Schedule the given job run, the scheduled job is handled by workload operator based resources available."""
+        if not self.middletier_namespace:
+            raise ConfigurationError(
+                "Unable to schedule package extract job without middletier namespace being set"
+            )
+
+        job_id = job_id or self._generate_id("package-analyzer")
+        parameters = locals()
```

Comment:
Won't be `job_id` repeated in the run method? (move assignemnt to `parameters` before assignment of to `job_id` environment variable)

Reply:
@fridex I definitely can move ```parameters``` before ```job_id```. However, I just took a look how other schedulers are written and it seems like all of them have ```job_id``` above ```parameters```. WDYT?

Reply:
Agh, right - you are right. The reference to `job_id` is always present. Keep it as is. Nice catch! :+1:

Reply:
Thanks a lot @fridex !